projects/pgai/pgai/configuration.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import Literal
+
+
+@dataclass
+class EmbeddingConfig:
+    model: str
+    dimensions: int
+    chat_user: str | None = None
+    api_key_name: str | None = None
+
+    
+@dataclass
+class ChunkingConfig:
```

Comment:
I'm only using one right now, because the recursive character text splitter is essentially the same as the character one if you only provide one splitting point right?
Maybe still good to support both for completeness.

Reply:
yeah, I'm also just wondering what will happen from a naming convention when we add a third chunking strategy